src/schema/re_frame.cljs

```diff
@@ -5,6 +5,16 @@
 (def ^:private ?cofx
   [:map])
 
+(def ^:private ?event-fx
+  [:maybe
+   [:map {:closed true}
+    [:db {:optional true} [:maybe map?]]
+    [:fx {:optional true}
+     [:maybe
```

Comment:
Even if re-frame's `:fx` understands nil, in practice, aren't we always passing at least an empty vector to it? In our production code I couldn't find a single instance where `:fx` is nil, we always pass a vector with zero or more elements or nothing at all (optional). So it looks like we can drop the `maybe` schema here too.

Reply:
I added `:maybe` for `:fx` and `:db` to be less restrictive, so don't have any examples from the codebase. I think if it's smth we want to avoid, will just keep the optional and remove the maybe.